zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/SessionsServiceImpl.java

```diff
@@ -21,29 +18,27 @@
 
 import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
 import static com.github.onsdigital.logging.v2.event.SimpleEvent.warn;
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.error;
 
 /**
  * Created by david on 12/03/2015.
  *
- * @deprecated In favour of the new JWT based sessions. Once the migration to the dp-identity-api is complete this class
- *             will be removed.
+ * @deprecated In favour of the transition ThreadLocalSessionsServiceImpl. The ThreadLocalSessionsServiceImpl should be
+ *             used in all cases. This calss
```

Comment:
Looks like there is something missing here

```suggestion
 *             used in all cases. This class will be removed.
```